timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/database/tinkerpop/TinkerPopOperations.java

```diff
@@ -1209,10 +1209,21 @@ public void addPredicateValueTypeVertexToVre(Vre vre) {
   @Override
   public void assertProperty(Vre vre, String rdfUri, RdfProperty property) {
     Vertex vertex = assertEntity(vre, rdfUri);
+    Object value;
+    try {
+      // The TinkerPopConverter does not need a collection to convert the property to the right db type.
```

Comment:
The TinkerPopConverter does not need a collection for the external -> db case. It does need one for the db -> external case (but we're not using that one here :)